README.md

```diff
@@ -5,7 +5,15 @@ Don't Slack Evil is the submission of the team "Les Gars" for the [2020 Slack Ap
 # Getting started
 
 1) Setup the [serverless framework](https://github.com/serverless/serverless)
-2) Deploy: `make deploy`
+2) Set up shared AWS credentials:
+  - Add Cyril for the shared AWS credentials
```

Comment:
? je suis pas sûr de comprendre la phrase ?

Reply:
*Ask I'll update the type